_commands/os_x/ditto.md

```diff
@@ -0,0 +1,72 @@
+---
+---
+
+Ditto
+-------
+
+Ditto is an OSX command that copies the entire contents of a working directory to another, new location.
+
+~~~ bash
+ditto /old/directory/ /new/directory/
+~~~
+
+<!--more-->
+
+### Details
+Ditto perserves file resources forks and metadata, in addition to ownership attributes. This ensures a more exact copy than cp. Ditto can also be used to copy a file or direct to a source that doesn't exist - in that case, ditto will automatically create the source for the user.
+
+
+### Useful Options / Examples
+
+#### -V 
+~~~ bash
+ditto -V old new
+>>>copying old
+copying file ./bla.txt ...
+10 bytes for ./bla.txt
```

Comment:
what's going on here is not intuitive. use "tree" or "ls" to show the contents of each directory before and after calling ditto. 

also, use backtacks " ` " around the work "ditto" throughout the document